config/webpack/partial/stats.js

```diff
@@ -6,7 +6,8 @@ var StatsWriterPlugin = archetype.devRequire("webpack-stats-plugin").StatsWriter
 
 module.exports = function (opts) {
   var statsOptions = {
-    filename: "../server/stats.json"
+    filename: "../server/stats.json",
+    fields: ['assetsByChunkName', 'assets']
```

Comment:
We now include `assets` so we can check if `manfiest.json` exists and conditionally load the SW stuff if it does